src/features/smartSearch/components/sankeyDiagram/SmartSearchSankeyProvider.tsx

```diff
@@ -0,0 +1,82 @@
+import { createContext, FC, ReactNode, useContext } from 'react';
+
+import makeSankeySegments from './makeSankeySegments';
+import useSmartSearchStats from 'features/smartSearch/hooks/useSmartSearchStats';
+import { ZetkinSmartSearchFilter } from '../types';
+import { SankeyConfig, SankeySegment } from './types';
+
+type SmartSearchSankeyProviderProps = {
+  arrowDepth?: number;
+  arrowWidth?: number;
+  children: ReactNode;
+  color?: string;
+  diagWidth?: number;
+  filters: ZetkinSmartSearchFilter[];
+  hoverColor?: string;
+  margin?: number;
+};
+
+type SankeyContextValue = {
+  config: SankeyConfig;
+  segments: SankeySegment[];
+};
+
+const SankeyContext = createContext<SankeyContextValue | null>(null);
+
+const SmartSearchSankeyProvider: FC<SmartSearchSankeyProviderProps> = ({
+  arrowDepth = 10,
+  arrowWidth = 20,
+  children,
+  color = '#cccccc',
+  diagWidth = 200,
+  hoverColor = '#bbbbbb',
```

Comment:
theme color

Reply:
Same here as with it's sibling `color`, but I will change it!